mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -260,53 +269,102 @@ export const AddOrRemoveCollateralForm = memo(({ route }: Props): JSX.Element =>
-        'flex-grow h-7': Platform.OS !== 'web',
-        'w-full': Platform.OS === 'web'
-      }])}
-      >
-        <ThemedText style={tailwind('mr-2')}>{translate('components/AddOrRemoveCollateralForm', 'Vault %')}</ThemedText>
-        <ThemedView
-          style={tailwind('flex flex-row items-center mb-0 py-1 px-1.5 rounded-2xl')}
-        >
-          <SymbolIcon
-            symbol={token.displaySymbol}
-          />
-          {displayNA
-            ? (
-              <ThemedText
-                light={tailwind('text-gray-900')}
-                dark={tailwind('text-gray-50')}
-                style={tailwind('px-1 text-sm font-medium')}
-                testID='bottom-sheet-vault-percentage-text'
-              >{translate('components/AddOrRemoveCollateralForm', 'N/A')}
-              </ThemedText>
-            )
-            : (
+      {isFeatureAvailable('dusd_vault_share')
+        ? (
+          <View
+            style={tailwind('flex justify-between items-center flex-row w-full')}
+          >
+            <ThemedText style={tailwind('mr-2 w-6/12')}>{translate('components/AddOrRemoveCollateralForm', 'Vault requirement')}</ThemedText>
+
+            <ThemedView
+              style={tailwind('flex flex-row items-center mb-0 py-1 px-1.5 rounded-2xl')}
+            >
+              <TokenIconGroup
+                testID='required_collateral_token_group'
+                symbols={requiredVaultShareTokens}
+                maxIconToDisplay={2}
+              />
               <NumberFormat
-                value={vaultValue}
+                value={requiredTokensShare.toFixed(2)}
                 thousandSeparator
                 decimalScale={2}
                 displayType='text'
                 suffix='%'
                 renderText={(val: string) => (
                   <ThemedView
-                    style={tailwind('px-1 rounded')}
+                    style={tailwind('flex flex-row px-1 rounded')}
                   >
                     <ThemedText
-                      light={tailwind('text-gray-900')}
-                      dark={tailwind('text-gray-50')}
+                      light={tailwind(['text-gray-900', { 'text-red-500': !isValidCollateralRatio }])}
+                      dark={tailwind(['text-gray-50', { 'text-red-500': !isValidCollateralRatio }])}
                       style={tailwind('text-sm font-medium')}
-                      testID='bottom-sheet-vault-percentage-text'
+                      testID='bottom-sheet-vault-requirement-text'
                     >
                       {val}
                     </ThemedText>
+                    <Text>{' '}</Text>
+                    <ThemedText
+                      dark={tailwind('text-gray-400')}
+                      light={tailwind('text-gray-500')}
+                      style={tailwind('text-sm font-medium')}
+                    >
+                      /50%
```

Comment:
Store this on a variable so in case we change this in the future

Reply:
Will update the logic